packages/core/src/Button/Button.styles.ts

```diff
@@ -45,82 +38,105 @@ export const { staticClasses, useClasses } = createClasses("HvButton", {
   },
   focusVisible: {},
   disabled: {
-    color: theme.colors.secondary_60,
-    borderColor: theme.colors.atmo4,
-    backgroundColor: theme.colors.atmo3,
     cursor: "not-allowed",
+    color: theme.colors.secondary_60,
+    borderColor: "transparent",
+    backgroundColor: "transparent",
+    "&:hover": {
+      backgroundColor: "transparent",
+      borderColor: "transparent",
+    },
+    "&:focus-visible": {
+      backgroundColor: "transparent",
+      borderColor: "transparent",
+    },
```

Comment:
Can't we merge these too? 👀 

```suggestion
    "&:hover, &:focus-visible": {
      backgroundColor: "transparent",
      borderColor: "transparent",
    },
```

Reply:
Of course, I'll update the styles 🙏

Reply:
Done ✅